sfputil/main.py

```diff
@@ -466,7 +466,13 @@ def presence(port):
 
         for physical_port in physical_port_list:
             port_name = get_physical_port_name(logical_port_name, i, ganged)
-            presence = platform_sfputil.get_presence(physical_port)
+
+            try:
+                presence = platform_sfputil.get_presence(physical_port)
+            except NotImplementedError:
+                click.echo("This functionality is currently not implemented for this platform")
+                sys.exit(6)
```

Comment:
all not implemented should return same exit code, easily for user to know the error.